CONTRIBUTING.rst

```diff
@@ -230,13 +236,45 @@ You should configure VS code as follow :
 
 
 
-Running test units
-~~~~~~~~~~~~~~~~~~~~~~
+Code quality control and testing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Every code change must be tested !
 
-DISDRODB tests are written using the third-party `pytest <https://docs.pytest.org>`_ package.
 
+To maintain a high code quality, we use `pre-commit <https://pre-commit.com/>`__ to run a series of checks on the code before each commit. The checks are defined in the .pre-commit-config.yaml file. The checks include:
+
```

Comment:
precommit does not include testing and coverage stuff !
Split precommit stuffs from `"testing and code coverage tool"